Makefile.shared

```diff
@@ -21,8 +21,11 @@ BUILD_WEBKIT_OPTIONS += --no-use-workspace
 BUILD_GOAL = $(notdir $(CURDIR))
 endif
 
+ifneq (,$(EXPORT_COMPILE_COMMANDS))
+	BUILD_WEBKIT_OPTIONS += --export-compile-commands
+endif
 
-XCODE_OPTIONS = `perl -I$(SCRIPTS_PATH) -Mwebkitdirs -e 'print XcodeOptionString()' -- $(BUILD_WEBKIT_OPTIONS)` $(ARGS)
+XCODE_OPTIONS = $(shell perl -I$(SCRIPTS_PATH) -Mwebkitdirs -e 'print XcodeOptionString()' -- $(BUILD_WEBKIT_OPTIONS)) $(ARGS)
```

Comment:
Is this necessary? I can't see why it would be.

Some internal build workflows rely on this evaluating _after_ the call to set-webkit-configuration, so I think it needs to stay deferred until the `xcodebuild` command line.

Reply:
It is a hack to avoid an unwanted strip of quotation marks. When we put XCODE_OPTIONS with the XcodeOptionString call directly to xcodebuild below, xcodebuild complains. `xcodebuild: error: invalid option '-gen-cdb-fragment-path'`
Maybe that is because xcode interprets
```
"OTHER_CFLAGS=$(inherited) -gen-cdb-fragment-path $(BUILT_PRODUCTS_DIR)/compile_commands"
```
to
```
OTHER_CFLAGS=$(inherited) -gen-cdb-fragment-path $(BUILT_PRODUCTS_DIR)/compile_commands
```
Any idea?